packages/tabs/src/TabTitle.tsx

```diff
@@ -125,6 +131,23 @@ const textOverflowStyles = css`
   text-overflow: ellipsis;
 `;
 
+function useDocumentActiveElement() {
+  const [activeEl, setActiveEl] = useState<Element | null>(null);
+
+  const handleFocusIn = useCallback(() => {
```

Comment:
Why do we need `useCallback` here? I don't think it's doing anything because we're only adding an event listener when the component mounts rather than creating one every render.

Reply:
Removed